programs/cpi-example/tests/dynamic_amm_init_pool.rs

```diff
@@ -137,15 +137,15 @@ async fn test_initialize_customizable_permissionless_pool() {
     test.add_program("metaplex", METAPLEX_PROGRAM_ID, None);
 
     let jup_vault = setup_vault_from_cluster(&mut test, JUP, mock_user.pubkey()).await;
-    let sol_vault = setup_vault_from_cluster(&mut test, USDC, mock_user.pubkey()).await;
+    let usdc_vault = setup_vault_from_cluster(&mut test, USDC, mock_user.pubkey()).await;
 
     let (mut banks_client, _, _) = test.start().await;
 
     let pool_key = derive_customizable_permissionless_constant_product_pool_key(JUP, USDC);
     let lp_mint = derive_lp_mint(pool_key);
 
     let a_vault_lp = derive_vault_lp_key(jup_vault.key, pool_key);
```

Comment:
- I think we should make helper to a common lib, so integration know how to derive all accounts in meteora 
- Make some function like: get_accounts_for_initialize_customizable_permissionless_pool() 
- For SOL/USDC/USDT in mainnet, we should make lp_vault hardcoded, caused it is not PDA in mainnet (many integrators are struggled when integrating with that)

Reply:
Fixed. Could you review the related PRs too?
https://github.com/mercurial-finance/mercurial-dynamic-amm-sdk/pull/195
https://github.com/MeteoraAg/stake-for-fee-sdk/pull/28
Thanks